pkgs/tools/system/plan9port/default.nix

```diff
@@ -19,26 +19,67 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace bin/9c \
       --replace 'which uniq' '${which}/bin/which uniq'
```

Comment:
If `which` is included as a build input (therefore, it can be found in the `$PATH`), then what is the meaning of this line?

Reply:
So `9c` can be used to build things other than p9p. See [osxsnarf] for an in-tree example.

[osxsnarf]: https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/os-specific/darwin/osxsnarf/default.nix